ci/release/update-version.sh

```diff
@@ -72,15 +72,17 @@ DEPENDENCIES=(
   ucx-py
   rapids-dask-dependency
 )
-for DEP in "${DEPENDENCIES[@]}"; do
-  for FILE in dependencies.yaml conda/environments/*.yaml python/cugraph-{pyg,dgl}/conda/*.yaml; do
+for FILE in dependencies.yaml conda/environments/*.yaml python/cugraph-{pyg,dgl}/conda/*.yaml; do
```

Comment:
I switched the iteration order of this double-for loop. That way we only find-replace ucx-py one time instead of once for every _other_ dependency.